Frameworks/Foundation/NSURLConnection.mm

```diff
@@ -396,6 +389,13 @@ - (void)URLProtocol:(NSURLProtocol*)urlProtocol didFailWithError:(NSError*)error
 - (void)URLProtocol:(NSURLProtocol*)urlProtocol
     didReceiveResponse:(NSURLResponse*)response
     cacheStoragePolicy:(NSURLCacheStoragePolicy)policy {
+    auto delegateCount = std::make_shared<std::atomic<uint>>(std::max(_scheduledRunLoops.size(), 1u));
+
+    __block StrongId<NSCachedURLResponse> cachedResponse;
```

Comment:
`__block StrongId<T>` will not commute ownership of the object into the block. Prefer `T*`.